src/azul/__init__.py

```diff
@@ -57,6 +57,14 @@ def debug(self, debug: int):
     def _validate_debug(self, debug):
         require(debug in (0, 1, 2), "AZUL_DEBUG must be either 0, 1 or 2")
 
+    @property
+    def debug_host(self):
+        return os.environ['AZUL_DEBUG_HOST']
```

Comment:
Combine into one variable called AZUL_REMOTE_DEBUG_ENDPOINT and use host:port notation